wpeverest/user-registration
# UR-1491 Dev - Compatibility temporarily disable users.

### All Submissions:






### Changes proposed in this Pull Request:



Previously there was no feature to disable user. This PR introduce a new feature to disable users.

### How to test the changes in this Pull Request:

1.Check in `view users` enable/ disable user if disabled ensure disabled user cannot login in wp-dashboard or frontend and if already logined user should be logout
2.Check same from bulk action.
3.Ensure enabled/Disabled status is displaying properly in user list table.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Compatibility temporarily disable users.
